### ISSUE TITLE
v13.5 - New Swedish translation "crashes" Umbraco

### ISSUE THREAD

**Issue by madsoulswe** [umbraco/Umbraco-CMS#17060]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13.5

### Bug summary

New Swedish translations in Umbraco v13.5 causes exception (including the front)
The translation file has two alias="apps" and LocalizedTextService.GetAreaStoredTranslations doesn't have any validation if overallResult has the key so it throws exception.

### Steps to reproduce

Install umbraco using swedish
Add document-types with properties e.g. Dropdown .
Render the property on front.

### Expected result / actual result

- Not crash
- Maybe add checks for duplicates
- Logg not crash.
- Add tests for all translation-files?

## Example of Exception
```
ArgumentException: An item with the same key has already been added. Key: apps
System.Collections.Generic.Dictionary<TKey, TValue>.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
Umbraco.Cms.Core.Services.LocalizedTextService.GetAreaStoredTranslations(IDictionary<CultureInfo, Lazy<XDocument>> xmlSource, CultureInfo cult)
Umbraco.Cms.Core.Services.LocalizedTextService+<>c__DisplayClass22_1.<XmlSourcesToAreaDictionary>b__0()
System.Lazy<T>.ViaFactory(LazyThreadSafetyMode mode)
System.Lazy<T>.ExecutionAndPublication(LazyHelper executionAndPublication, bool useDefaultConstructor)
System.Lazy<T>.CreateValue()
System.Lazy<T>.get_Value()
Umbraco.Cms.Core.Services.LocalizedTextService.GetFromDictionarySource(CultureInfo culture, string area, string key, IDictionary<string, string> tokens)
Umbraco.Cms.Core.Services.LocalizedTextService.Localize(string area, string alias, CultureInfo culture, IDictionary<string, string> tokens)
Umbraco.Extensions.LocalizedTextServiceExtensions.Localize(ILocalizedTextService manager, string area, string alias)
Umbraco.Cms.Core.PropertyEditors.DropDownFlexibleConfigurationEditor..ctor(ILocalizedTextService textService, IIOHelper ioHelper, IEditorConfigurationParser editorConfigurationParser)
Umbraco.Cms.Core.PropertyEditors.DropDownFlexiblePropertyEditor.CreateConfigurationEditor()
Umbraco.Cms.Core.PropertyEditors.DataEditor.GetConfigurationEditor()
Umbraco.Cms.Core.Models.DataType+<>c__DisplayClass19_0.<GetLazyConfiguration>b__1()
```

**Comment by madsoulswe** [comment]
PR #17059

**Comment by madsoulswe** [comment]
@jrunestone Yes I agree. LocalizedTextService hade tests, but not with the embed xmls. But it shouldnt fail anymore.

I will add another PR with some basic xml-format tests of the embeded xmls, and a refactoring and optimizations of  LocalizedTextService. 

I have also included a "LocalizedTextService-test" that loads all xmls but I'm not sure what should be tested. 😅

https://github.com/madsoulswe/Umbraco-CMS/blob/47f5ea5835d9a48e03e732e8116bf6a28ada5f95/tests/Umbraco.Tests.UnitTests/Umbraco.Infrastructure/Services/LocalizedTextServiceTests.cs#L383

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/17059 - will be available in v13.5.1
